diff --git a/node_modules/fastmcp/dist/FastMCP.js b/node_modules/fastmcp/dist/FastMCP.js
index 8b131b7..a5cb799 100644
--- a/node_modules/fastmcp/dist/FastMCP.js
+++ b/node_modules/fastmcp/dist/FastMCP.js
@@ -390,6 +390,7 @@ var FastMCPSession = class extends FastMCPSessionEventEmitter {
       this.#capabilities.prompts = {};
     }
     this.#capabilities.logging = {};
+    this.#capabilities.completions = {};
     this.#server = new Server(
       { name, version },
       { capabilities: this.#capabilities, instructions }
